services/main/Controllers/Tag.cs

```diff
@@ -19,11 +22,17 @@ public TagsController(ILogger<MediaController> logger, SaveTags saveTagsService,
         _searchTagsService = searchTagService;
     }
 
-    [HttpPost]
-    public OutputResponse<Models.Tag> Create([FromBody] CreateTagInput input)
+    [Authorize]
+    [HttpPost("create/tag")]

```

Comment:
```suggestion
    [HttpPost]

```

You can learn somethings about API DESIGN here: https://swagger.io/resources/articles/best-practices-in-api-design/

Reply:
thank you boss